### PR TITLE
Mobility

### PR DESCRIPTION
**Pull request by gidden**
Adds the section about working at work and home.

### CODE REVIEW

**Comment by gidden** [comment]
@gonuke and @scopatz, this is the content we discussed about working from home and work. I'm not sure how or if we want to incorporate it. If you get a chance, could you give it a quick glance and let me know your thoughts?

File: version-control/git/mobility/Readme.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+(i.e., coding project collaboration) and thesis writing (i.e., "personal"
+activities).
+
+The workflow in this section describes three repository locations - a server, a
+work computer, and a home (laptop) computer. The server will host the "base"
+repository, and the server can live anywhere you have a connection to. For
+example, you could use your GitHub repository (described in the
+[Collaborate](../remote/Readme.md) section as the server. If you have access to
+a server on campus (e.g., I have user space on CAE's server
+@best-tux.cae.wisc.edu), you can host your repository there.
```
Comment (scopatz) [inline version-control/git/mobility/Readme.md:21]: I know that this is for a UW bootcamp, but you might want to steer clear of UW specific elements if you want to PR this back into the main SWC bootcamp repo.

**Comment by gidden** [comment]
I'll merge this as well, as neither remote nor local links to it. If @scopatz or @gonuke think it is a good idea to do so, I'll let you link when you develop the related content.

File: version-control/git/mobility/Readme.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+[website](http://www.johndcook.com/blog/2008/04/03/microsoft-word-and-latex/)
+that explains the difference.
+
+![wordvlatex](https://raw.github.com/gidden/boot-camps/mobility/version-control/git/mobility/wordvslatex.gif "Word vs. Latex")
+
+I'll add that, from my experience, the types of documents needed for the left
+side of the curve (where Word is easier than Latex) are simple enough to just
+use Google Docs (which is version controlled as well!). 
+
+Furthermore, I simply **can't imagine** having to write something as complicated
```
Comment (scopatz) [inline version-control/git/mobility/Readme.md:146]: Rephase to use less "I" statements.

File: version-control/git/mobility/Readme.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+Have you ever struggled with formatting Word's equations, chapters,
+bibliography, etc.? [Latex](http://www.latex-project.org/) works wonders with
+that. Here's a great graph taken from John Cook's
+[website](http://www.johndcook.com/blog/2008/04/03/microsoft-word-and-latex/)
+that explains the difference.
+
+![wordvlatex](https://raw.github.com/gidden/boot-camps/mobility/version-control/git/mobility/wordvslatex.gif "Word vs. Latex")
+
+I'll add that, from my experience, the types of documents needed for the left
```
Comment (scopatz) [inline version-control/git/mobility/Readme.md:142]: Rephase to use less "I" statements.  "Experience shows..."

**Comment by scopatz** [comment]
Overall it looks good.  There are a couple of spots where it reads more like a blog than training material.  I would rephrase these to be more about the students and their potential for pain than you and the things that you have learn from past pain.
